addons/hearing/functions/fnc_firedNear.sqf

```diff
@@ -47,10 +47,18 @@ if (_silencer != "") then {
     _audibleFireCoef = getNumber (configFile >> "CfgWeapons" >> _silencer >> "ItemInfo" >> "AmmoCoef" >> "audibleFire");
 };
 
-_audibleFire = getNumber (configFile >> "CfgAmmo" >> _ammo >> "audibleFire");
-
-_loudness = _audibleFireCoef * _audibleFire / 64;
-_strength = _vehAttenuation * (_loudness - (_loudness/50 * _distance)); // linear drop off
+_magazine = (getArray(configFile >> "CfgWeapons" >> _weapon >> "magazines")) select 0;
```

Comment:
This line is a bit fishy. The first magazine supported by the weapon may not be the one that fired, which may be even more true when magazineGroups are used.